tests/test_optim.py

```diff
@@ -531,7 +531,9 @@ def test_step_sizes_fixed_parameters_minuit(mocker):
 
     assert minuit.called
     assert minimizer.fixed == [True, False, False]
-    assert minimizer.errors == [0.0, 0.01, 0.01]
+    # iminuit v2.12.2+ sets non-zero step sizes for all parameters including fixed parameters
+    # c.f. https://github.com/scikit-hep/iminuit/issues/762
+    assert minimizer.errors == [0.01, 0.01, 0.01]
```

Comment:
@lukasheinrich @kratsg I'm for now sidestepping this by just accepting that `iminuit` has its step size behavior that we need to deal with and that the minimizer step size and the fit result uncertainties might not match. I'm also setting the `v2.12.2` lower bound to make it so this isn't confusing between versions. Let me know if you find this unacceptable.

Reply:
Though maybe given @alexander-held's comment https://github.com/scikit-hep/pyhf/pull/1919#pullrequestreview-1053689055 it would be better to just say that `iminuit` will control the step size and not have this test?

Reply:
I think this test can be removed, it previously checked that the zero values were set properly for fixed parameters, but now this test no longer checks any `pyhf` behavior (only `iminuit` defaults).